### ISSUE TITLE
Value of collected ads wrong when pages change title dynamically

### ISSUE THREAD

**Issue by mneunomne** [dhowe/AdNauseam#2247]
Sometimes the value displayed on the extension badge is not correct, showing 0 instead of the actual number of collected/visited ads. 

![Screen Shot 2022-12-30 at 10 03 27](https://user-images.githubusercontent.com/4967860/210039694-de39cbe2-8d2b-45a0-a2f6-23ba59c6cacd.png)

Not sure how to reproduce yet, but it can occasionally happen. Will investigate more.

**Comment by dhowe** [comment]
Isn't that correct behavior for a page with no ads?

**Comment by mneunomne** [comment]
Ah true yes, I always get confused with this. Closing

**Comment by mneunomne** [comment]
Actually, not only the page title changes, but the url itself, and you scroll through the website above. Don't think there is anything we can do about that. uBlock manages not to have the same issue.

**Comment by mneunomne** [comment]
@dhowe the question is: 

Do we want to number on the badge to display the collected ads a specific page, or specific domain? Maybe domain makes more sense than page?

**Comment by dhowe** [comment]
For me page makes more sense, but I'm open to discussion as domain could work as well. But the question of time frame is not clear to me: what about visits to a domain from a previous session or a previous year (we might have many thousands of ads listed for, say, a search results page)? Would also involve a fair amount of implementation changes (eg scanning the admap for domain entries within a time window for each new ad detected), for what seems at best a minimal gain in user experience.

Regardless, this question does not relate to the issue in the ticket above, which I have marked as `Won't fix` since the  assumption throughout the code base is that a URL change is, by definition, a page change.

Closing for now, but feel free to open a new ticket or discussion on the issue above, or the count-per-domain question if you think its warrants further discussion
